doc/ApiDocGeneration/Generate-DocIndex.ps1

```diff
@@ -0,0 +1,140 @@
+    "datashare"="Data Share"; "deploymentmanager"="Deployment Manager";
+    "deviceprovisioningservices"="Device Provisioning Services"; "devspaces"="Dev Spaces";
+    "Dev Test Labs"=""; "dns"="DNS";
+    "edgegateway"="Edge Gateway";
+    "eventgrid"="Event Grid"; "eventhub"="Event Hub";
+    "frontdoor"="Front Door"; "graphrbac"="Active Directory";
+    "guestconfiguration"="Guest Configuration"; "hdinsight"="HDInsight";
+    "healthcareapis"="Healthcare APIs"; "hybridcompute"="Hybrid Cloud";
+    "hybriddatamanager"="Hybrid Datamanager"; "identity"="Identity";
+    "insights"="Insights"; "iotcentral"="IoT Central";
+    "iothub"="IoT"; "keyvault"="Keyvault";
+    "kusto"="Kusto"; "labservices"="LabServices";
+    "locationbasedservices"="Location Based Services"; "logic"="Logic Apps";
+    "machinelearning"="Machine Learning"; "machinelearningcompute"="Machine Learning Compute";
+    "maintenance"="Maintenance"; "managednetwork"="Managed Network";
+    "managedserviceidentity"="Managed Service Identity"; "managedservices"="Managed Services";
+    "managementgroups"="Management Groups"; "managementpartner"="Management Partner";
+    "maps"="Maps"; "marketplaceordering"="Market Place Ordering";
+    "mediaservices"="Media Services";
+    "mixedreality"="Mixed Reality"; "monitor"="Monitor";
+    "netapp"="NetApp"; "network"="Network";
+    "notificationhubs"="Notification Hubs"; "operationalinsights"="Operational Insights";
+    "peering"="Peering"; "policyinsights"="Policy Insights";
+    "postgresql"="PostgreSQL"; "powerbidedicated"="PowerBI Dedicated";
+    "privatedns"="Private DNS"; "recoveryservices"="Recovery Services";
+    "recoveryservices-backup"="Recovery Services Backup"; "recoveryservices-siterecovery"="Recovery Services Site Recovery";
+    "redis"="Redis"; "relay"="Relay";
+    "reservations"="Reservations"; "resourcegraph"="Resource Graph";
+    "resources"="Resources"; "scheduler"="Scheduler";
+    "search"="Search"; "securitycenter"="Security Center";
+    "servermanagement"="Server Management"; "servicebus"="Service Bus";
+    "servicefabric"="Service Fabric"; "signalr"="signalR";
+    "sqlmanagement"="SQL Management"; "sqlvirtualmachine"="SQL Virtual Machine";
+    "storage"="Storage"; "storagecache"="Storage Cache";
+    "storagesync"="Storage Sync"; "storsimple"="Stor Simple";
+    "storsimple8000series"="Stor Simple 8000 series"; "streamanalytics"="Stream Analytics";
+    "subscription"="Subscription";
+    "trafficmanager"="Traffic Manager"; "websites"="Websites";
+}
+
+
+Write-Verbose "Initializing Default DocFx Site..."
+& "$($RepoRoot)/docfx/docfx.exe" init -q -o "$($RepoRoot)/docfx_project"
+
+Write-Verbose "Copying template and configuration..."
+New-Item -Path "$($RepoRoot)/docfx_project" -Name "templates" -ItemType "directory"
+Copy-Item "$($RepoRoot)/doc/ApiDocGeneration/templates/*" -Destination "$($RepoRoot)/docfx_project/templates" -Force -Recurse
```

Comment:
What about these paths? It seems like they are still assuming the ApiDocGeneration path?

Reply:
Same below. We should either pass the path in or compute it relative to where this file lives.

Reply:
Fixed now. Passed it as an argument to the script.